presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -16,7 +28,30 @@ running queries.
 When the server is configured to use HTTPS, a :doc:`password authenticator
 </develop/password-authenticator>` such as :doc:`LDAP </security/ldap>` or
 :doc:`password file </security/password-file>` must be configured in order to
-use the Web UI.
+use the Web UI. In these cases, the authentication type is automatically
+defaulted to the required ``form`` value. It triggers the Web UI to display a
+login form that accepts the relevant username and password and verifies against
+the correct authentication backend.
```

Comment:
I find these sections confusing.  There are now multiple authentication systems supported, so I feel `must` is too strong here.  Also in the following sections is says `also supports usage of certificate authentication`, which to me sounds like I can use form and certificate, but these are alternatives.

Reply:
Updated.